server/db-helpers/index.js

```diff
@@ -31,6 +31,7 @@ function manuscriptGqlToDb(manuscript, owner) {
 function manuscriptDbToGql(manuscriptDb, id) {
   const manuscript = _.cloneDeep(manuscriptDb)
   manuscript.id = id
+  delete manuscript.type
```

Comment:
What does this do?

Reply:
Converts from DB schema to GraphQL schema :) there's no type in the GraphQL schema. This will help when we'll end up using Manuscript object for the business logic.

Reply:
Why do we have 'manuscript type' in the GraphQL schema but not the DB schema? Does 'type' refer to research article/feature article/etc?

Reply:
Just saw this, it's the other way around. This function converts from a DB object to a GraphQL object. All DB objects have a type field since the only table is `entities` and you need a way to know the entity type (e.g. User, Manuscript etc). So no, it doesn't refer to research/feature article.